CocoaHeadsApp/CocoaHeadsApp/AppDelegate.swift

```diff
@@ -6,10 +6,13 @@ let logger = XCGLogger.defaultInstance()
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    var window: UIWindow?
+    var appCoordinator: AppCoordinator!
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         
+        self.appCoordinator = AppCoordinator()
+        self.appCoordinator.start()
+        
```

Comment:
Acho que podemos usar o `self` somente quando realmente for preciso. Caso contrário, omitimos.

Se alguém tiver alguma opinião contrária, é só dizer.

Talvez dê pra colocar isso no lint depois (ainda preciso ver se tem isso nas regras).